react-native-purchases-ui/ios/RNPaywalls.m

```diff
@@ -58,21 +58,20 @@ - (PaywallProxy *)paywalls API_AVAILABLE(ios(15.0)){
 
 // MARK: -
 
-RCT_EXPORT_METHOD(presentPaywall:(NSDictionary *)options
+RCT_EXPORT_METHOD(presentPaywall:(nullable NSString *)offeringIdentifier
+                  shouldDisplayCloseButton:(BOOL)displayCloseButton
```

Comment:
I think typed parameters are safer so I replaced the map with the real parameters now that we don't need a nullable boolean `displayCloseButton` in iOS, which was what made me add this map parameter